docs/content/en/docs/getting-started/_index.md

```diff
@@ -1,594 +1,5 @@
 ---
-title: Getting started with Keptn Observability
```

Comment:
Do we need to keep this page if it's empty ?

Reply:
It's the index for the section so yes, it is needed to control where this section plays out.  I would like to have the main section pages have some content that explains what is in the section but other people don't like this and keep deleting the content.  So we need to discuss this as part of the bigger project to define what the docs look like.